### DISCUSSION TITLE
Support @ as a tag symbol like jnrl

### DISCUSSION THREAD

**Discussion by jgarte** [localauthor/zk#78] [Ideas]
Hi,

What do you think if we support `@` as a tag symbol like jnrl:

https://jrnl.sh/en/stable/overview/#tags

![image](https://github.com/localauthor/zk/assets/47760695/64899a9d-50bd-40cf-b99e-12372388b0a7)

**Comment by localauthor** [comment]
See 'zk-tag-regexp'. Have you tried setting that?
